.github/workflows/build.yaml

```diff
@@ -0,0 +1,17 @@
+name: Build
+on: [push]
```

Comment:
Do we want this CI to run on a push to any branch, or only a push to the main branch?  Either is fine with me.  Paul, you can decide.

Reply:
cc @cuddihyge

Reply:
I would recommend that we do on every push, so that developers can test how their code will do in CI before opening a merge request and sending a notification to all the people who "Watch" (in Github terms) this repo. Plus, it doesn't cost us anything :smile: